workspaces/lluis/Content.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="typescript">
+  export let additionalClasses = ""
```

Comment:
hmm, I don't love the fact that this component takes `additionalClasses`. 

I can see that this is only used in one place, can you perhaps think of a way of avoiding that usage? As long as it doesn't break anything, feel free to change other things in order to achieve this.